evap/results/tools.py

```diff
@@ -228,36 +228,37 @@ def _get_results_impl(evaluation: Evaluation, *, refetch_related_objects: bool =
     prefetch_related_objects([evaluation], *GET_RESULTS_PREFETCH_LOOKUPS)
 
     tas_per_contribution_question: dict[tuple[int, int], list[TextAnswer]] = unordered_groupby(
-        ((textanswer.contribution_id, textanswer.question_id), textanswer)
```

Comment:
I feel we should be thorough and rename variables like this one to disambiguate assignments and questions. It would be best if we could also do it in the tests (I read something like `cls.contributor_questions = baker.make(QuestionAssignment, ...)`), but if it is too much work we can also consider doing it in a follow up. Outside of tests though, we should do it right away